### DISCUSSION TITLE
Global Settings for multi-user Windows environment

### DISCUSSION THREAD

**Discussion by BDaddyG** [keepassxreboot/keepassxc#6109] [Q&A] [answered]
Hello

Running Keepass-XC in a multi-user RDP/Citrix Windows environment.

Is there any way of setting (and locking) settings globally (for all users)?

For example, disabling the "Check for updates" dialog at first run, disabling update checks, and preventing users from enabling it?  

Maybe by Group Policy or a global settings file.

Thanks!

**Comment by droidmonkey** [comment, marked as answer]
Not yet, planned though! See #2189

**Reply by BDaddyG** [reply]
Thanks for the info!  I'll keep a watch for the update that implements this.
